src/com/oltpbenchmark/WorkloadConfiguration.java

```diff
@@ -358,6 +364,15 @@ public void setMaxLoaderRetries(int maxLoaderRetries) {
   public int getMaxLoaderRetries() {
     return maxLoaderRetries;
   }
+  
+  // Geo partitioned options.
+  public boolean getGeoPartitioningEnabled() {
+      return geoPartitionPolicy.isPartitioningEnabled();
```

Comment:
Is not it much more simpler to use null for `geoPartitionPolicy` member in case it is not enabled? For now API seems strange. It is possible that `getGeoPartitioningEnabled` return false but `getGeoPartitioningPolicy` returns not null object. How it can be used in this case?
From my point of view single function is enough
```
public GeoPartitionPolicy getGeoPartitioningPolicy() {
    return geoPartitionPolicy;
}
```

Reply:
Done